docs/source/DeveloperDocumentation/TaskEngineWorkflows.md

```diff
@@ -839,6 +839,143 @@ Job callbacks are triggered when the entire job has completed. Below is a list o
 | message           | List of files to requested for deletion the destination storage. |
 | custom_data       | Returns the custom data submitted to the workflow. |
 
+## MEDIATAILOR CHANNEL ASSEMBLY
+
+This workflow allows for creating and updating live compose streams - very similar to [VOD REMIX](TaskEngineWorkflows.html#vod-remix) [Live Compose](TaskEngineWorkflowFeatures.html#avod-and-live-compose). Please refer to [Live Compose with Manifest Manipulation](TaskEngineWorkflowFeatures.html#live-compose-with-manifest-manipulation) for more information and ad break singaling examples.
```

Comment:
"live compose" should probably be consistent (same with the "live/Live" comments), and also be a capitalised term "Live Compose" within our documentation as it is a concept/feature name.